certified-connectors/ZellisIntelligencePlatform/README.md

```diff
@@ -0,0 +1,44 @@
+## Zellis Intelligence Platform Connector
+An official connector working with the Zellis Intelligence Platform.
+
+## Prerequisites
+You will need the following to proceed:
+* A Microsoft Power Apps or Power Automate plan
```

Comment:
@basha-k The connection type of Your connector is OAUTH, it is not mentioned in the "Prerequisites" section here. Please provide detailed steps for how to obtain OAUTH credentials.